deployments/common/crds-v1beta1/appprotect.f5.com_appolicies.yaml

```diff
@@ -1058,6 +1199,8 @@ spec:
                 urls:
                   items:
                     properties:
+                      $action:
```

Comment:
just to double check, is `$` in front of the action is correct?

Reply:
yes, it means that this field should be deleted from the base policy rather than added